app/views/projects/show.html.erb

```diff
@@ -0,0 +1,5 @@
+  <p>Project: <%= @project.name %></p>
+  <p>Material: <%= @project.material%>
+  <p>Challenge Theme: <%= @project.challenge.theme %></p>
+  <p>Number of Contestants: <%= @project.contestants.count %></p>
```

Comment:
We don't want to use `.count` in the view because that is technically a data calculation. While it is simple, the best most MVC compliant place for it is in a model method. Similar to what you've done with average years of experience, you can encapsulate `contestants.count` in a method called `contestant_count` in the Project model